Essentials/src/main/java/com/earth2me/essentials/Essentials.java

```diff
@@ -329,6 +332,9 @@ public void onEnable() {
                     knownCommandsProvider = new ReflKnownCommandsProvider();
                 }
 
+                //Command Alias provider
+                formattedCommandAliasProvider = new ReflFormattedCommandAliasProvider(PaperLib.isPaper());
```

Comment:
is there a reason you didn't make a paper specific provider? also has this been tested on 1.8.8-1.16.4?

Reply:
Yup! should be working just fine on all implementations of paper (saw that the modification to FormattedCommandAliasProvider was added in to paper before the 1.8 launch), and its still present and works fine on the latest 1.16.4 version.

I will update the class to make separate classes for both servertypes!

Reply:
Splitted the file up in a paper and spigot version, however, due to the need for the reflection library I also had to include the NMSReflectionProvider in the PaperProvider module -- not sure if that would be a problem?